raceSystem/templates/home.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Unicycle race system</title>
+  <style>
+    #leftBanner{
+      position: relative;
+      float: left;
+      height: 100%;
+      width: 15%;
+    }
+    #frameTracks{
+      position: relative;
+      height: 100%;
+      width: 100%;
+    }
+    #rightBanner{
+      float: right;
+      height: 100%;
+      width: 15%;
+    }
+    #track_1{
+      position: relative;
+      float: left;
+      left:10px;
+      border: 3px solid black;
+      width: 33%;
+      height: 600px;
+      margin-top: 25px;
+      margin-bottom: 25px;
+      text-align: center;   /*cemter text horizontal*/
+      line-height: 600px;   /*center text vertical*/
+      font-size: 30px;      /*text size*/
+    }
+    #startLine_1{
+      position: absolute;
+      bottom: 15px;
+      border: 2px dotted red;
+      width: 100%;
+      height: 10px;
+    }
+    #finishLine_1{
+      position: absolute;
+      top:      15px;
+      border:   2px dotted rgb(4, 0, 255);
+      width: 100%;
+      height: 10px;
+    }
+    #track_2{
+      position: relative;
+      float: right;
+      right:20px;
+      border: 3px solid black;
+      width: 33%;
+      height: 600px;
+      margin-top: 25px;
+      margin-bottom: 25px;
+      text-align: center;   /*cemter text horizontal*/
+      line-height: 600px;   /*center text vertical*/
+      font-size: 30px;      /*text size*/
+    }
+    #startLine_2{
+      position: absolute;
+      bottom: 15px;
+      border: 2px dotted red;
+      width: 100%;
+      height: 10px;
+    }
+    #finishLine_2{
+      position: absolute;
+      top:      15px;
+      border:   2px dotted rgb(4, 0, 255);
+      width: 100%;
+      height: 10px;
+    }
+
+    .raceResult{
+      position: absolute;
+      top:20%;
+      width: 100%;
+      height: 80%;
+      font-size: medium;
+    }
+    
+  </style>
+  <script>
+    function updateRaceResult(sRaceResult_1,sRaceResult_2){
+      var raceResult_1 = document.getElementById("raceResult_1")
+      var raceResult_2 = document.getElementById("raceResult_2")
+
+      raceResult_1.innerHTML = sRaceResult_1
+      raceResult_2.innerHTML = sRaceResult_2
+    }
+
+    function receivedStatus(xStartClientConnected,xFinishClientConnected,sRaceStatus,sRaceResult_1,sRaceResult_2){
+      updateClientConnectionStatus(xStartClientConnected, xFinishClientConnected)
+      updateRaceStatus(sText = sRaceStatus)
+      updateRaceResult(sRaceResult_1,sRaceResult_2)
+    }
+
+    setInterval(function(){
+        fetch('/get-status')
+        .then(function(response) {
+          return response.json();
+        })
+        .then(function(data) {
+          receivedStatus(data.xStartClientConnected, data.xFinishClientConnected,data.sRaceStatus,data.sRaceResult_1,data.sRaceResult_2);
+        });
+      }, 1000);
```

Comment:
This magic number is the nr of ms for the refresh rate? Can you define it in a variable? (let refreshRateInMs = 1000) and put it at the top of this script block

Reply:
Done, exactly as requested.